### ISSUE TITLE
    fatal error: lipo: can't map input file:     /Users/eb/AndroidStudioProjects/playpadel/build/ios/Debug-iphonesimulator/Ru     nner.app/Frameworks/App.framework/flutter_assets/ios/Flutter/App.framework/     (Invalid argument)

### ISSUE THREAD

**Issue by ioplaypadelapp** [flutter/flutter#64259]
## Steps to Reproduce

Something weird is happening. 
Flutter can't build ios app unless every time i have to build i do the following:
- move ios folder to bin (using system and not android studio)
- run in terminal: 
-- flutter clean
-- flutter create.
-- flutter run

## Logs

<details> 
<summary>logs</summary> 

```

Last login: Thu Aug 20 15:11:55 on ttys003
eb@erics-MacBook-Pro playpadel % flutter create .                
Recreating project ....
  ios/Runner.xcworkspace/contents.xcworkspacedata (created)
  ios/Runner.xcworkspace/xcshareddata/IDEWorkspaceChecks.plist (created)
  ios/Runner.xcworkspace/xcshareddata/WorkspaceSettings.xcsettings (created)
  ios/Runner/Info.plist (created)
  ios/Runner/Assets.xcassets/LaunchImage.imageset/LaunchImage@2x.png (created)
  ios/Runner/Assets.xcassets/LaunchImage.imageset/LaunchImage@3x.png (created)
  ios/Runner/Assets.xcassets/LaunchImage.imageset/README.md (created)
  ios/Runner/Assets.xcassets/LaunchImage.imageset/Contents.json (created)
  ios/Runner/Assets.xcassets/LaunchImage.imageset/LaunchImage.png (created)
  ios/Runner/Assets.xcassets/AppIcon.appiconset/Icon-App-76x76@2x.png (created)
  ios/Runner/Assets.xcassets/AppIcon.appiconset/Icon-App-29x29@1x.png (created)
  ios/Runner/Assets.xcassets/AppIcon.appiconset/Icon-App-40x40@1x.png (created)
  ios/Runner/Assets.xcassets/AppIcon.appiconset/Icon-App-20x20@1x.png (created)
  ios/Runner/Assets.xcassets/AppIcon.appiconset/Icon-App-1024x1024@1x.png
  (created)
  ios/Runner/Assets.xcassets/AppIcon.appiconset/Icon-App-83.5x83.5@2x.png
  (created)
  ios/Runner/Assets.xcassets/AppIcon.appiconset/Icon-App-20x20@3x.png (created)
  ios/Runner/Assets.xcassets/AppIcon.appiconset/Contents.json (created)
  ios/Runner/Assets.xcassets/AppIcon.appiconset/Icon-App-20x20@2x.png (created)
  ios/Runner/Assets.xcassets/AppIcon.appiconset/Icon-App-29x29@3x.png (created)
  ios/Runner/Assets.xcassets/AppIcon.appiconset/Icon-App-40x40@2x.png (created)
  ios/Runner/Assets.xcassets/AppIcon.appiconset/Icon-App-60x60@3x.png (created)
  ios/Runner/Assets.xcassets/AppIcon.appiconset/Icon-App-60x60@2x.png (created)
  ios/Runner/Assets.xcassets/AppIcon.appiconset/Icon-App-76x76@1x.png (created)
  ios/Runner/Assets.xcassets/AppIcon.appiconset/Icon-App-40x40@3x.png (created)
  ios/Runner/Assets.xcassets/AppIcon.appiconset/Icon-App-29x29@2x.png (created)
  ios/Runner/Base.lproj/LaunchScreen.storyboard (created)
  ios/Runner/Base.lproj/Main.storyboard (created)
  ios/Runner.xcodeproj/project.xcworkspace/contents.xcworkspacedata (created)
  ios/Runner.xcodeproj/project.xcworkspace/xcshareddata/IDEWorkspaceChecks.plist
  (created)
  ios/Runner.xcodeproj/project.xcworkspace/xcshareddata/WorkspaceSettings.xcsett
  ings (created)
  ios/Runner.xcodeproj/xcshareddata/xcschemes/Runner.xcscheme (created)
  ios/Flutter/Debug.xcconfig (created)
  ios/Flutter/Release.xcconfig (created)
  ios/Flutter/AppFrameworkInfo.plist (created)
  ios/.gitignore (created)
  ios/Runner/Runner-Bridging-Header.h (created)
  ios/Runner/AppDelegate.swift (created)
  ios/Runner.xcodeproj/project.pbxproj (created)
 
Wrote 41 files.                             

All done!
[✓] Flutter: is fully installed. (Channel stable, 1.20.2, on Mac OS X 10.15.6
    19G2021, locale en-IT)
[✓] Android toolchain - develop for Android devices: is fully installed.
    (Android SDK version 30.0.2)
[✓] Xcode - develop for iOS and macOS: is fully installed. (Xcode 11.6)
[✓] Android Studio: is fully installed. (version 4.0)
[✓] Connected device: is fully installed. (1 available)

In order to run your application, type:

  $ cd .
  $ flutter run

Your application code is in ./lib/main.dart.

eb@erics-MacBook-Pro playpadel % flutter clean
Cleaning Xcode workspace...                                         4.4s
Deleting build...                                                  299ms
Deleting .dart_tool...                                               2ms
Deleting Generated.xcconfig...                                       1ms
Deleting flutter_export_environment.sh...                            0ms
eb@erics-MacBook-Pro playpadel % flutter run
Launching lib/main.dart on iPhone SE (2nd generation) in debug mode...
 
Running pod install...                                             11.7s
Running Xcode build...                                                  
                                                   
 └─Compiling, linking and signing...                         6.9s
Xcode build done.                                           34.0s
Failed to build iOS app
Error output from Xcode build:
↳
    ** BUILD FAILED **


Xcode's output:
↳
    /Users/eb/flutter/.pub-cache/hosted/pub.dartlang.org/firebase_messaging-7.0.
    0/ios/Classes/FLTFirebaseMessagingPlugin.m:190:43: warning:
    'FIRMessagingRemoteMessage' is deprecated: FCM direct channel is deprecated,
    please use APNs for downstream message handling. [-Wdeprecated-declarations]
    - (void)applicationReceivedRemoteMessage:(FIRMessagingRemoteMessage
    *)remoteMessage {
                                              ^
    In module 'FirebaseMessaging' imported from
    /Users/eb/AndroidStudioProjects/playpadel/ios/Pods/Headers/Public/Firebase/F
    irebase.h:72:
    /Users/eb/AndroidStudioProjects/playpadel/ios/Pods/FirebaseMessaging/Firebas
    eMessaging/Sources/Public/FIRMessaging.h:219:1: note:
    'FIRMessagingRemoteMessage' has been explicitly marked deprecated here
    __deprecated_msg(
    ^
    In module 'UIKit' imported from
    /Users/eb/AndroidStudioProjects/playpadel/ios/Pods/Target Support
    Files/firebase_messaging/firebase_messaging-prefix.pch:2:
    In module 'Foundation' imported from
    /Applications/Xcode.app/Contents/Developer/Platforms/iPhoneSimulator.platfor
    m/Developer/SDKs/iPhoneSimulator13.6.sdk/System/Library/Frameworks/UIKit.fra
    mework/Headers/UIKit.h:8:
    In module 'CoreFoundation' imported from
    /Applications/Xcode.app/Contents/Developer/Platforms/iPhoneSimulator.platfor
    m/Developer/SDKs/iPhoneSimulator13.6.sdk/System/Library/Frameworks/Foundatio
    n.framework/Headers/Foundation.h:6:
    In module 'Darwin' imported from
    /Applications/Xcode.app/Contents/Developer/Platforms/iPhoneSimulator.platfor
    m/Developer/SDKs/iPhoneSimulator13.6.sdk/System/Library/Frameworks/CoreFound
    ation.framework/Headers/CoreFoundation.h:16:
    /Applications/Xcode.app/Contents/Developer/Platforms/iPhoneSimulator.platfor
    m/Developer/SDKs/iPhoneSimulator13.6.sdk/usr/include/sys/cdefs.h:191:48:
    note: expanded from macro '__deprecated_msg'
            #define __deprecated_msg(_msg) __attribute__((__deprecated__(_msg)))
                                                          ^
    /Users/eb/flutter/.pub-cache/hosted/pub.dartlang.org/firebase_messaging-7.0.
    0/ios/Classes/FLTFirebaseMessagingPlugin.m:290:24: warning:
    'FIRMessagingRemoteMessage' is deprecated: FCM direct channel is deprecated,
    please use APNs for downstream message handling. [-Wdeprecated-declarations]
        didReceiveMessage:(FIRMessagingRemoteMessage *)remoteMessage {
                           ^
    In module 'FirebaseMessaging' imported from
    /Users/eb/AndroidStudioProjects/playpadel/ios/Pods/Headers/Public/Firebase/F
    irebase.h:72:
    /Users/eb/AndroidStudioProjects/playpadel/ios/Pods/FirebaseMessaging/Firebas
    eMessaging/Sources/Public/FIRMessaging.h:219:1: note:
    'FIRMessagingRemoteMessage' has been explicitly marked deprecated here
    __deprecated_msg(
    ^
    In module 'UIKit' imported from
    /Users/eb/AndroidStudioProjects/playpadel/ios/Pods/Target Support
    Files/firebase_messaging/firebase_messaging-prefix.pch:2:
    In module 'Foundation' imported from
    /Applications/Xcode.app/Contents/Developer/Platforms/iPhoneSimulator.platfor
    m/Developer/SDKs/iPhoneSimulator13.6.sdk/System/Library/Frameworks/UIKit.fra
    mework/Headers/UIKit.h:8:
    In module 'CoreFoundation' imported from
    /Applications/Xcode.app/Contents/Developer/Platforms/iPhoneSimulator.platfor
    m/Developer/SDKs/iPhoneSimulator13.6.sdk/System/Library/Frameworks/Foundatio
    n.framework/Headers/Foundation.h:6:
    In module 'Darwin' imported from
    /Applications/Xcode.app/Contents/Developer/Platforms/iPhoneSimulator.platfor
    m/Developer/SDKs/iPhoneSimulator13.6.sdk/System/Library/Frameworks/CoreFound
    ation.framework/Headers/CoreFoundation.h:16:
    /Applications/Xcode.app/Contents/Developer/Platforms/iPhoneSimulator.platfor
    m/Developer/SDKs/iPhoneSimulator13.6.sdk/usr/include/sys/cdefs.h:191:48:
    note: expanded from macro '__deprecated_msg'
            #define __deprecated_msg(_msg) __attribute__((__deprecated__(_msg)))
                                                          ^
    /Users/eb/flutter/.pub-cache/hosted/pub.dartlang.org/firebase_messaging-7.0.
    0/ios/Classes/FLTFirebaseMessagingPlugin.m:136:30: warning:
    'shouldEstablishDirectChannel' is deprecated: FCM direct channel is
    deprecated, please use APNs channel for downstream message delivery.
    [-Wdeprecated-declarations]
        [FIRMessaging messaging].shouldEstablishDirectChannel = true;
                                 ^
    In module 'FirebaseMessaging' imported from
    /Users/eb/AndroidStudioProjects/playpadel/ios/Pods/Headers/Public/Firebase/F
    irebase.h:72:
    /Users/eb/AndroidStudioProjects/playpadel/ios/Pods/FirebaseMessaging/Firebas
    eMessaging/Sources/Public/FIRMessaging.h:283:56: note:
    'shouldEstablishDirectChannel' has been explicitly marked deprecated here
    @property(nonatomic) BOOL shouldEstablishDirectChannel
    DEPRECATED_MSG_ATTRIBUTE(
                                                           ^
    In module 'UIKit' imported from
    /Users/eb/AndroidStudioProjects/playpadel/ios/Pods/Target Support
    Files/firebase_messaging/firebase_messaging-prefix.pch:2:
    In module 'Foundation' imported from
    /Applications/Xcode.app/Contents/Developer/Platforms/iPhoneSimulator.platfor
    m/Developer/SDKs/iPhoneSimulator13.6.sdk/System/Library/Frameworks/UIKit.fra
    mework/Headers/UIKit.h:8:
    In module 'CoreFoundation' imported from
    /Applications/Xcode.app/Contents/Developer/Platforms/iPhoneSimulator.platfor
    m/Developer/SDKs/iPhoneSimulator13.6.sdk/System/Library/Frameworks/Foundatio
    n.framework/Headers/Foundation.h:6:
    In module 'Darwin' imported from
    /Applications/Xcode.app/Contents/Developer/Platforms/iPhoneSimulator.platfor
    m/Developer/SDKs/iPhoneSimulator13.6.sdk/System/Library/Frameworks/CoreFound
    ation.framework/Headers/CoreFoundation.h:16:
    /Applications/Xcode.app/Contents/Developer/Platforms/iPhoneSimulator.platfor
    m/Developer/SDKs/iPhoneSimulator13.6.sdk/usr/include/AvailabilityMacros.h:18
    2:64: note: expanded from macro 'DEPRECATED_MSG_ATTRIBUTE'
                #define DEPRECATED_MSG_ATTRIBUTE(s)
                __attribute__((deprecated(s)))
                                                                   ^
    /Users/eb/flutter/.pub-cache/hosted/pub.dartlang.org/firebase_messaging-7.0.
    0/ios/Classes/FLTFirebaseMessagingPlugin.m:191:52: warning: 'appData' is
    deprecated [-Wdeprecated-declarations]
      [self didReceiveRemoteNotification:remoteMessage.appData];
                                                       ^
    In module 'FirebaseMessaging' imported from
    /Users/eb/AndroidStudioProjects/playpadel/ios/Pods/Headers/Public/Firebase/F
    irebase.h:72:
    /Users/eb/AndroidStudioProjects/playpadel/ios/Pods/FirebaseMessaging/Firebas
    eMessaging/Sources/Public/FIRMessaging.h:226:62: note: 'appData' has been
    explicitly marked deprecated here
    @property(nonatomic, readonly, strong) NSDictionary *appData
    DEPRECATED_ATTRIBUTE;
                                                                 ^
    In module 'UIKit' imported from
    /Users/eb/AndroidStudioProjects/playpadel/ios/Pods/Target Support
    Files/firebase_messaging/firebase_messaging-prefix.pch:2:
    In module 'Foundation' imported from
    /Applications/Xcode.app/Contents/Developer/Platforms/iPhoneSimulator.platfor
    m/Developer/SDKs/iPhoneSimulator13.6.sdk/System/Library/Frameworks/UIKit.fra
    mework/Headers/UIKit.h:8:
    In module 'CoreFoundation' imported from
    /Applications/Xcode.app/Contents/Developer/Platforms/iPhoneSimulator.platfor
    m/Developer/SDKs/iPhoneSimulator13.6.sdk/System/Library/Frameworks/Foundatio
    n.framework/Headers/Foundation.h:6:
    In module 'Darwin' imported from
    /Applications/Xcode.app/Contents/Developer/Platforms/iPhoneSimulator.platfor
    m/Developer/SDKs/iPhoneSimulator13.6.sdk/System/Library/Frameworks/CoreFound
    ation.framework/Headers/CoreFoundation.h:16:
    /Applications/Xcode.app/Contents/Developer/Platforms/iPhoneSimulator.platfor
    m/Developer/SDKs/iPhoneSimulator13.6.sdk/usr/include/AvailabilityMacros.h:18
    0:60: note: expanded from macro 'DEPRECATED_ATTRIBUTE'
            #define DEPRECATED_ATTRIBUTE        __attribute__((deprecated))
                                                               ^
    /Users/eb/flutter/.pub-cache/hosted/pub.dartlang.org/firebase_messaging-7.0.
    0/ios/Classes/FLTFirebaseMessagingPlugin.m:289:1: warning: implementing
    deprecated method [-Wdeprecated-implementations]
    - (void)messaging:(FIRMessaging *)messaging
    ^
    In module 'FirebaseMessaging' imported from
    /Users/eb/AndroidStudioProjects/playpadel/ios/Pods/Headers/Public/Firebase/F
    irebase.h:72:
    /Users/eb/AndroidStudioProjects/playpadel/ios/Pods/FirebaseMessaging/Firebas
    eMessaging/Sources/Public/FIRMessaging.h:253:1: note: method
    'messaging:didReceiveMessage:' declared here
    - (void)messaging:(FIRMessaging *)messaging
    ^
    /Users/eb/flutter/.pub-cache/hosted/pub.dartlang.org/firebase_messaging-7.0.
    0/ios/Classes/FLTFirebaseMessagingPlugin.m:291:63: warning: 'appData' is
    deprecated [-Wdeprecated-declarations]
      [_channel invokeMethod:@"onMessage" arguments:remoteMessage.appData];
                                                                  ^
    In module 'FirebaseMessaging' imported from
    /Users/eb/AndroidStudioProjects/playpadel/ios/Pods/Headers/Public/Firebase/F
    irebase.h:72:
    /Users/eb/AndroidStudioProjects/playpadel/ios/Pods/FirebaseMessaging/Firebas
    eMessaging/Sources/Public/FIRMessaging.h:226:62: note: 'appData' has been
    explicitly marked deprecated here
    @property(nonatomic, readonly, strong) NSDictionary *appData
    DEPRECATED_ATTRIBUTE;
                                                                 ^
    In module 'UIKit' imported from
    /Users/eb/AndroidStudioProjects/playpadel/ios/Pods/Target Support
    Files/firebase_messaging/firebase_messaging-prefix.pch:2:
    In module 'Foundation' imported from
    /Applications/Xcode.app/Contents/Developer/Platforms/iPhoneSimulator.platfor
    m/Developer/SDKs/iPhoneSimulator13.6.sdk/System/Library/Frameworks/UIKit.fra
    mework/Headers/UIKit.h:8:
    In module 'CoreFoundation' imported from
    /Applications/Xcode.app/Contents/Developer/Platforms/iPhoneSimulator.platfor
    m/Developer/SDKs/iPhoneSimulator13.6.sdk/System/Library/Frameworks/Foundatio
    n.framework/Headers/Foundation.h:6:
    In module 'Darwin' imported from
    /Applications/Xcode.app/Contents/Developer/Platforms/iPhoneSimulator.platfor
    m/Developer/SDKs/iPhoneSimulator13.6.sdk/System/Library/Frameworks/CoreFound
    ation.framework/Headers/CoreFoundation.h:16:
    /Applications/Xcode.app/Contents/Developer/Platforms/iPhoneSimulator.platfor
    m/Developer/SDKs/iPhoneSimulator13.6.sdk/usr/include/AvailabilityMacros.h:18
    0:60: note: expanded from macro 'DEPRECATED_ATTRIBUTE'
            #define DEPRECATED_ATTRIBUTE        __attribute__((deprecated))
                                                               ^
    6 warnings generated.
    building file list ... done
    deleting App.framework/_CodeSignature/CodeResources
    deleting App.framework/_CodeSignature/
    App.framework/
    App.framework/App
    App.framework/flutter_assets/AssetManifest.json
    App.framework/flutter_assets/FontManifest.json
    App.framework/flutter_assets/NOTICES
    App.framework/flutter_assets/kernel_blob.bin
    App.framework/flutter_assets/ios/Flutter/App.framework/flutter_assets/assets
    /logo.png
    App.framework/flutter_assets/ios/Flutter/App.framework/flutter_assets/images
    /logo.png
    App.framework/flutter_assets/ios/Flutter/App.framework/flutter_assets/ios/Fl
    utter/App.framework/flutter_assets/
    App.framework/flutter_assets/ios/Flutter/App.framework/flutter_assets/ios/Fl
    utter/App.framework/flutter_assets/images/logo.png
    App.framework/flutter_assets/ios/Flutter/App.framework/flutter_assets/ios/Fl
    utter/App.framework/flutter_assets/ios/
    App.framework/flutter_assets/ios/Flutter/App.framework/flutter_assets/ios/Fl
    utter/App.framework/flutter_assets/ios/Flutter/
    App.framework/flutter_assets/ios/Flutter/App.framework/flutter_assets/ios/Fl
    utter/App.framework/flutter_assets/ios/Flutter/App.framework/
    App.framework/flutter_assets/ios/Flutter/App.framework/flutter_assets/ios/Fl
    utter/App.framework/flutter_assets/ios/Flutter/App.framework/flutter_assets/
    App.framework/flutter_assets/ios/Flutter/App.framework/flutter_assets/ios/Fl
    utter/App.framework/flutter_assets/ios/Flutter/App.framework/flutter_assets/
    assets/
    App.framework/flutter_assets/ios/Flutter/App.framework/flutter_assets/ios/Fl
    utter/App.framework/flutter_assets/ios/Flutter/App.framework/flutter_assets/
    assets/logo.png
    App.framework/flutter_assets/ios/Flutter/App.framework/flutter_assets/ios/Fl
    utter/App.framework/flutter_assets/ios/Flutter/App.framework/flutter_assets/
    images/
    App.framework/flutter_assets/ios/Flutter/App.framework/flutter_assets/ios/Fl
    utter/App.framework/flutter_assets/ios/Flutter/App.framework/flutter_assets/
    images/logo.png

    sent 34340272 bytes  received 288 bytes  68681120.00 bytes/sec
    total size is 39882976  speedup is 1.16
    building file list ... done
    Flutter.framework/
    Flutter.framework/Flutter
    Flutter.framework/Info.plist
    Flutter.framework/icudtl.dat
    Flutter.framework/_CodeSignature/
    Flutter.framework/_CodeSignature/CodeResources

    sent 98828341 bytes  received 120 bytes  65885640.67 bytes/sec
    total size is 98815875  speedup is 1.00
    /Users/eb/AndroidStudioProjects/playpadel/build/ios/Debug-iphonesimulator/Ru
    nner.app/Frameworks/App.framework/App: signed bundle with Mach-O universal
    (x86_64 arm64) [io.flutter.flutter.app]
    /Users/eb/AndroidStudioProjects/playpadel/build/ios/Debug-iphonesimulator/Ru
    nner.app/Frameworks/Flutter.framework/Flutter: replacing existing signature
    /Users/eb/AndroidStudioProjects/playpadel/build/ios/Debug-iphonesimulator/Ru
    nner.app/Frameworks/Flutter.framework/Flutter: signed bundle with Mach-O
    universal (armv7 x86_64 arm64) [io.flutter.flutter]
    2020-08-20 16:23:58.534 defaults[79678:652241] 
    The domain/default pair of
    (/Users/eb/AndroidStudioProjects/playpadel/build/ios/Debug-iphonesimulator/R
    unner.app/Frameworks/App.framework/flutter_assets/ios/Flutter/App.framework/
    Info.plist, CFBundleExecutable) does not exist
    fatal error: lipo: can't map input file:
    /Users/eb/AndroidStudioProjects/playpadel/build/ios/Debug-iphonesimulator/Ru
    nner.app/Frameworks/App.framework/flutter_assets/ios/Flutter/App.framework/
    (Invalid argument)
    fatal error: lipo: can't map input file:
    /Users/eb/AndroidStudioProjects/playpadel/build/ios/Debug-iphonesimulator/Ru
    nner.app/Frameworks/App.framework/flutter_assets/ios/Flutter/App.framework/
    (Invalid argument)
    Failed to extract x86_64 for
    /Users/eb/AndroidStudioProjects/playpadel/build/ios/Debug-iphonesimulator/Ru
    nner.app/Frameworks/App.framework/flutter_assets/ios/Flutter/App.framework/.
    Running lipo -info:
    fatal error: lipo: can't map input file:
    /Users/eb/AndroidStudioProjects/playpadel/build/ios/Debug-iphonesimulator/Ru
    nner.app/Frameworks/App.framework/flutter_assets/ios/Flutter/App.framework/
    (Invalid argument)
    Command PhaseScriptExecution failed with a nonzero exit code
    note: Using new build system
    note: Building targets in parallel
    note: Planning build
    note: Constructing build description
    warning: Capabilities for Signing & Capabilities may not function correctly
    because its entitlements use a placeholder team ID. To resolve this, select
    a development team in the Runner editor. (in target 'Runner' from project
    'Runner')

Could not build the application for the simulator.
Error launching application on iPhone SE (2nd generation).
eb@erics-MacBook-Pro playpadel % flutter clean
Deleting build...                                                  272ms
Deleting .dart_tool...                                               6ms
eb@erics-MacBook-Pro playpadel % flutter create .
Recreating project ....
  ios/Runner.xcworkspace/contents.xcworkspacedata (created)
  ios/Runner.xcworkspace/xcshareddata/IDEWorkspaceChecks.plist (created)
  ios/Runner.xcworkspace/xcshareddata/WorkspaceSettings.xcsettings (created)
  ios/Runner/Info.plist (created)
  ios/Runner/Assets.xcassets/LaunchImage.imageset/LaunchImage@2x.png (created)
  ios/Runner/Assets.xcassets/LaunchImage.imageset/LaunchImage@3x.png (created)
  ios/Runner/Assets.xcassets/LaunchImage.imageset/README.md (created)
  ios/Runner/Assets.xcassets/LaunchImage.imageset/Contents.json (created)
  ios/Runner/Assets.xcassets/LaunchImage.imageset/LaunchImage.png (created)
  ios/Runner/Assets.xcassets/AppIcon.appiconset/Icon-App-76x76@2x.png (created)
  ios/Runner/Assets.xcassets/AppIcon.appiconset/Icon-App-29x29@1x.png (created)
  ios/Runner/Assets.xcassets/AppIcon.appiconset/Icon-App-40x40@1x.png (created)
  ios/Runner/Assets.xcassets/AppIcon.appiconset/Icon-App-20x20@1x.png (created)
  ios/Runner/Assets.xcassets/AppIcon.appiconset/Icon-App-1024x1024@1x.png (created)
  ios/Runner/Assets.xcassets/AppIcon.appiconset/Icon-App-83.5x83.5@2x.png (created)
  ios/Runner/Assets.xcassets/AppIcon.appiconset/Icon-App-20x20@3x.png (created)
  ios/Runner/Assets.xcassets/AppIcon.appiconset/Contents.json (created)
  ios/Runner/Assets.xcassets/AppIcon.appiconset/Icon-App-20x20@2x.png (created)
  ios/Runner/Assets.xcassets/AppIcon.appiconset/Icon-App-29x29@3x.png (created)
  ios/Runner/Assets.xcassets/AppIcon.appiconset/Icon-App-40x40@2x.png (created)
  ios/Runner/Assets.xcassets/AppIcon.appiconset/Icon-App-60x60@3x.png (created)
  ios/Runner/Assets.xcassets/AppIcon.appiconset/Icon-App-60x60@2x.png (created)
  ios/Runner/Assets.xcassets/AppIcon.appiconset/Icon-App-76x76@1x.png (created)
  ios/Runner/Assets.xcassets/AppIcon.appiconset/Icon-App-40x40@3x.png (created)
  ios/Runner/Assets.xcassets/AppIcon.appiconset/Icon-App-29x29@2x.png (created)
  ios/Runner/Base.lproj/LaunchScreen.storyboard (created)
  ios/Runner/Base.lproj/Main.storyboard (created)
  ios/Runner.xcodeproj/project.xcworkspace/contents.xcworkspacedata (created)
  ios/Runner.xcodeproj/project.xcworkspace/xcshareddata/IDEWorkspaceChecks.plist (created)
  ios/Runner.xcodeproj/project.xcworkspace/xcshareddata/WorkspaceSettings.xcsettings (created)
  ios/Runner.xcodeproj/xcshareddata/xcschemes/Runner.xcscheme (created)
  ios/Flutter/Debug.xcconfig (created)
  ios/Flutter/Release.xcconfig (created)
  ios/Flutter/AppFrameworkInfo.plist (created)
  ios/.gitignore (created)
  ios/Runner/Runner-Bridging-Header.h (created)
  ios/Runner/AppDelegate.swift (created)
  ios/Runner.xcodeproj/project.pbxproj (created)
Running "flutter pub get" in playpadel...                           1.0s
 
Wrote 41 files.                             

All done!
[✓] Flutter: is fully installed. (Channel stable, 1.20.2, on Mac OS X 10.15.6 19G2021, locale en-IT)
[✓] Android toolchain - develop for Android devices: is fully installed. (Android SDK version 30.0.2)
[✓] Xcode - develop for iOS and macOS: is fully installed. (Xcode 11.6)
[✓] Android Studio: is fully installed. (version 4.0)
[✓] Connected device: is fully installed. (1 available)

In order to run your application, type:

  $ cd .
  $ flutter run

Your application code is in ./lib/main.dart.

eb@erics-MacBook-Pro playpadel % run flutter
zsh: command not found: run
eb@erics-MacBook-Pro playpadel % flutter run
Launching lib/main.dart on iPhone SE (2nd generation) in debug mode...
 
Running pod install...                                             14.7s
Running Xcode build...                                                  
                                                   
 └─Compiling, linking and signing...                        20.3s
Xcode build done.                                           200.2s
Waiting for iPhone SE (2nd generation) to report its views...          3ms
Syncing files to device iPhone SE (2nd generation)...              319ms

Flutter run key commands.
r Hot reload. 🔥🔥🔥
R Hot restart.
h Repeat this help message.
d Detach (terminate "flutter run" but leave application running).
c Clear the screen
q Quit (terminate the application on the device).
An Observatory debugger and profiler on iPhone SE (2nd generation) is available at: http://127.0.0.1:54048/zaRhvgJ9LBg=/

Performing hot restart...                                               
Restarted application in 679ms.

  [Restored 20 Aug 2020 at 16:38:53]
Last login: Thu Aug 20 16:38:53 on ttys001
eb@erics-MacBook-Pro playpadel % flutter run
Launching lib/main.dart on iPhone SE (2nd generation) in debug mode...
 
Running Xcode build...                                                  
                                                   
 └─Compiling, linking and signing...                         8.0s
Xcode build done.                                           27.3s
Failed to build iOS app
Error output from Xcode build:
↳
    ** BUILD FAILED **


Xcode's output:
↳
    /Users/eb/flutter/.pub-cache/hosted/pub.dartlang.org/firebase_messaging-7.0.0/ios/Classes/FLTFirebaseMessagingPlugin.m:190:43:
    warning: 'FIRMessagingRemoteMessage' is deprecated: FCM direct channel is deprecated, please use APNs for downstream message
    handling. [-Wdeprecated-declarations]
    - (void)applicationReceivedRemoteMessage:(FIRMessagingRemoteMessage *)remoteMessage {
                                              ^
    In module 'FirebaseMessaging' imported from
    /Users/eb/AndroidStudioProjects/playpadel/ios/Pods/Headers/Public/Firebase/Firebase.h:72:
    /Users/eb/AndroidStudioProjects/playpadel/ios/Pods/FirebaseMessaging/FirebaseMessaging/Sources/Public/FIRMessaging.h:219:1: note:
    'FIRMessagingRemoteMessage' has been explicitly marked deprecated here
    __deprecated_msg(
    ^
    In module 'UIKit' imported from /Users/eb/AndroidStudioProjects/playpadel/ios/Pods/Target Support
    Files/firebase_messaging/firebase_messaging-prefix.pch:2:
    In module 'Foundation' imported from
    /Applications/Xcode.app/Contents/Developer/Platforms/iPhoneSimulator.platform/Developer/SDKs/iPhoneSimulator13.6.sdk/System/Library/
    Frameworks/UIKit.framework/Headers/UIKit.h:8:
    In module 'CoreFoundation' imported from
    /Applications/Xcode.app/Contents/Developer/Platforms/iPhoneSimulator.platform/Developer/SDKs/iPhoneSimulator13.6.sdk/System/Library/
    Frameworks/Foundation.framework/Headers/Foundation.h:6:
    In module 'Darwin' imported from
    /Applications/Xcode.app/Contents/Developer/Platforms/iPhoneSimulator.platform/Developer/SDKs/iPhoneSimulator13.6.sdk/System/Library/
    Frameworks/CoreFoundation.framework/Headers/CoreFoundation.h:16:
    /Applications/Xcode.app/Contents/Developer/Platforms/iPhoneSimulator.platform/Developer/SDKs/iPhoneSimulator13.6.sdk/usr/include/sys
    /cdefs.h:191:48: note: expanded from macro '__deprecated_msg'
            #define __deprecated_msg(_msg) __attribute__((__deprecated__(_msg)))
                                                          ^
    /Users/eb/flutter/.pub-cache/hosted/pub.dartlang.org/firebase_messaging-7.0.0/ios/Classes/FLTFirebaseMessagingPlugin.m:290:24:
    warning: 'FIRMessagingRemoteMessage' is deprecated: FCM direct channel is deprecated, please use APNs for downstream message
    handling. [-Wdeprecated-declarations]
        didReceiveMessage:(FIRMessagingRemoteMessage *)remoteMessage {
                           ^
    In module 'FirebaseMessaging' imported from
    /Users/eb/AndroidStudioProjects/playpadel/ios/Pods/Headers/Public/Firebase/Firebase.h:72:
    /Users/eb/AndroidStudioProjects/playpadel/ios/Pods/FirebaseMessaging/FirebaseMessaging/Sources/Public/FIRMessaging.h:219:1: note:
    'FIRMessagingRemoteMessage' has been explicitly marked deprecated here
    __deprecated_msg(
    ^
    In module 'UIKit' imported from /Users/eb/AndroidStudioProjects/playpadel/ios/Pods/Target Support
    Files/firebase_messaging/firebase_messaging-prefix.pch:2:
    In module 'Foundation' imported from
    /Applications/Xcode.app/Contents/Developer/Platforms/iPhoneSimulator.platform/Developer/SDKs/iPhoneSimulator13.6.sdk/System/Library/
    Frameworks/UIKit.framework/Headers/UIKit.h:8:
    In module 'CoreFoundation' imported from
    /Applications/Xcode.app/Contents/Developer/Platforms/iPhoneSimulator.platform/Developer/SDKs/iPhoneSimulator13.6.sdk/System/Library/
    Frameworks/Foundation.framework/Headers/Foundation.h:6:
    In module 'Darwin' imported from
    /Applications/Xcode.app/Contents/Developer/Platforms/iPhoneSimulator.platform/Developer/SDKs/iPhoneSimulator13.6.sdk/System/Library/
    Frameworks/CoreFoundation.framework/Headers/CoreFoundation.h:16:
    /Applications/Xcode.app/Contents/Developer/Platforms/iPhoneSimulator.platform/Developer/SDKs/iPhoneSimulator13.6.sdk/usr/include/sys
    /cdefs.h:191:48: note: expanded from macro '__deprecated_msg'
            #define __deprecated_msg(_msg) __attribute__((__deprecated__(_msg)))
                                                          ^
    /Users/eb/flutter/.pub-cache/hosted/pub.dartlang.org/firebase_messaging-7.0.0/ios/Classes/FLTFirebaseMessagingPlugin.m:136:30:
    warning: 'shouldEstablishDirectChannel' is deprecated: FCM direct channel is deprecated, please use APNs channel for downstream
    message delivery. [-Wdeprecated-declarations]
        [FIRMessaging messaging].shouldEstablishDirectChannel = true;
                                 ^
    In module 'FirebaseMessaging' imported from
    /Users/eb/AndroidStudioProjects/playpadel/ios/Pods/Headers/Public/Firebase/Firebase.h:72:
    /Users/eb/AndroidStudioProjects/playpadel/ios/Pods/FirebaseMessaging/FirebaseMessaging/Sources/Public/FIRMessaging.h:283:56: note:
    'shouldEstablishDirectChannel' has been explicitly marked deprecated here
    @property(nonatomic) BOOL shouldEstablishDirectChannel DEPRECATED_MSG_ATTRIBUTE(
                                                           ^
    In module 'UIKit' imported from /Users/eb/AndroidStudioProjects/playpadel/ios/Pods/Target Support
    Files/firebase_messaging/firebase_messaging-prefix.pch:2:
    In module 'Foundation' imported from
    /Applications/Xcode.app/Contents/Developer/Platforms/iPhoneSimulator.platform/Developer/SDKs/iPhoneSimulator13.6.sdk/System/Library/
    Frameworks/UIKit.framework/Headers/UIKit.h:8:
    In module 'CoreFoundation' imported from
    /Applications/Xcode.app/Contents/Developer/Platforms/iPhoneSimulator.platform/Developer/SDKs/iPhoneSimulator13.6.sdk/System/Library/
    Frameworks/Foundation.framework/Headers/Foundation.h:6:
    In module 'Darwin' imported from
    /Applications/Xcode.app/Contents/Developer/Platforms/iPhoneSimulator.platform/Developer/SDKs/iPhoneSimulator13.6.sdk/System/Library/
    Frameworks/CoreFoundation.framework/Headers/CoreFoundation.h:16:
    /Applications/Xcode.app/Contents/Developer/Platforms/iPhoneSimulator.platform/Developer/SDKs/iPhoneSimulator13.6.sdk/usr/include/Ava
    ilabilityMacros.h:182:64: note: expanded from macro 'DEPRECATED_MSG_ATTRIBUTE'
                #define DEPRECATED_MSG_ATTRIBUTE(s) __attribute__((deprecated(s)))
                                                                   ^
    /Users/eb/flutter/.pub-cache/hosted/pub.dartlang.org/firebase_messaging-7.0.0/ios/Classes/FLTFirebaseMessagingPlugin.m:191:52:
    warning: 'appData' is deprecated [-Wdeprecated-declarations]
      [self didReceiveRemoteNotification:remoteMessage.appData];
                                                       ^
    In module 'FirebaseMessaging' imported from
    /Users/eb/AndroidStudioProjects/playpadel/ios/Pods/Headers/Public/Firebase/Firebase.h:72:
    /Users/eb/AndroidStudioProjects/playpadel/ios/Pods/FirebaseMessaging/FirebaseMessaging/Sources/Public/FIRMessaging.h:226:62: note:
    'appData' has been explicitly marked deprecated here
    @property(nonatomic, readonly, strong) NSDictionary *appData DEPRECATED_ATTRIBUTE;
                                                                 ^
    In module 'UIKit' imported from /Users/eb/AndroidStudioProjects/playpadel/ios/Pods/Target Support
    Files/firebase_messaging/firebase_messaging-prefix.pch:2:
    In module 'Foundation' imported from
    /Applications/Xcode.app/Contents/Developer/Platforms/iPhoneSimulator.platform/Developer/SDKs/iPhoneSimulator13.6.sdk/System/Library/
    Frameworks/UIKit.framework/Headers/UIKit.h:8:
    In module 'CoreFoundation' imported from
    /Applications/Xcode.app/Contents/Developer/Platforms/iPhoneSimulator.platform/Developer/SDKs/iPhoneSimulator13.6.sdk/System/Library/
    Frameworks/Foundation.framework/Headers/Foundation.h:6:
    In module 'Darwin' imported from
    /Applications/Xcode.app/Contents/Developer/Platforms/iPhoneSimulator.platform/Developer/SDKs/iPhoneSimulator13.6.sdk/System/Library/
    Frameworks/CoreFoundation.framework/Headers/CoreFoundation.h:16:
    /Applications/Xcode.app/Contents/Developer/Platforms/iPhoneSimulator.platform/Developer/SDKs/iPhoneSimulator13.6.sdk/usr/include/Ava
    ilabilityMacros.h:180:60: note: expanded from macro 'DEPRECATED_ATTRIBUTE'
            #define DEPRECATED_ATTRIBUTE        __attribute__((deprecated))
                                                               ^
    /Users/eb/flutter/.pub-cache/hosted/pub.dartlang.org/firebase_messaging-7.0.0/ios/Classes/FLTFirebaseMessagingPlugin.m:289:1:
    warning: implementing deprecated method [-Wdeprecated-implementations]
    - (void)messaging:(FIRMessaging *)messaging
    ^
    In module 'FirebaseMessaging' imported from
    /Users/eb/AndroidStudioProjects/playpadel/ios/Pods/Headers/Public/Firebase/Firebase.h:72:
    /Users/eb/AndroidStudioProjects/playpadel/ios/Pods/FirebaseMessaging/FirebaseMessaging/Sources/Public/FIRMessaging.h:253:1: note:
    method 'messaging:didReceiveMessage:' declared here
    - (void)messaging:(FIRMessaging *)messaging
    ^
    /Users/eb/flutter/.pub-cache/hosted/pub.dartlang.org/firebase_messaging-7.0.0/ios/Classes/FLTFirebaseMessagingPlugin.m:291:63:
    warning: 'appData' is deprecated [-Wdeprecated-declarations]
      [_channel invokeMethod:@"onMessage" arguments:remoteMessage.appData];
                                                                  ^
    In module 'FirebaseMessaging' imported from
    /Users/eb/AndroidStudioProjects/playpadel/ios/Pods/Headers/Public/Firebase/Firebase.h:72:
    /Users/eb/AndroidStudioProjects/playpadel/ios/Pods/FirebaseMessaging/FirebaseMessaging/Sources/Public/FIRMessaging.h:226:62: note:
    'appData' has been explicitly marked deprecated here
    @property(nonatomic, readonly, strong) NSDictionary *appData DEPRECATED_ATTRIBUTE;
                                                                 ^
    In module 'UIKit' imported from /Users/eb/AndroidStudioProjects/playpadel/ios/Pods/Target Support
    Files/firebase_messaging/firebase_messaging-prefix.pch:2:
    In module 'Foundation' imported from
    /Applications/Xcode.app/Contents/Developer/Platforms/iPhoneSimulator.platform/Developer/SDKs/iPhoneSimulator13.6.sdk/System/Library/
    Frameworks/UIKit.framework/Headers/UIKit.h:8:
    In module 'CoreFoundation' imported from
    /Applications/Xcode.app/Contents/Developer/Platforms/iPhoneSimulator.platform/Developer/SDKs/iPhoneSimulator13.6.sdk/System/Library/
    Frameworks/Foundation.framework/Headers/Foundation.h:6:
    In module 'Darwin' imported from
    /Applications/Xcode.app/Contents/Developer/Platforms/iPhoneSimulator.platform/Developer/SDKs/iPhoneSimulator13.6.sdk/System/Library/
    Frameworks/CoreFoundation.framework/Headers/CoreFoundation.h:16:
    /Applications/Xcode.app/Contents/Developer/Platforms/iPhoneSimulator.platform/Developer/SDKs/iPhoneSimulator13.6.sdk/usr/include/Ava
    ilabilityMacros.h:180:60: note: expanded from macro 'DEPRECATED_ATTRIBUTE'
            #define DEPRECATED_ATTRIBUTE        __attribute__((deprecated))
                                                               ^
    6 warnings generated.
    building file list ... done
    deleting App.framework/_CodeSignature/CodeResources
    deleting App.framework/_CodeSignature/
    App.framework/
    App.framework/App
    App.framework/flutter_assets/
    App.framework/flutter_assets/AssetManifest.json
    App.framework/flutter_assets/FontManifest.json
    App.framework/flutter_assets/NOTICES
    App.framework/flutter_assets/ios/
    App.framework/flutter_assets/ios/Flutter/
    App.framework/flutter_assets/ios/Flutter/App.framework/
    App.framework/flutter_assets/ios/Flutter/App.framework/flutter_assets/
    App.framework/flutter_assets/ios/Flutter/App.framework/flutter_assets/assets/
    App.framework/flutter_assets/ios/Flutter/App.framework/flutter_assets/assets/logo.png
    App.framework/flutter_assets/ios/Flutter/App.framework/flutter_assets/images/
    App.framework/flutter_assets/ios/Flutter/App.framework/flutter_assets/images/logo.png

    sent 854119 bytes  received 200 bytes  1708638.00 bytes/sec
    total size is 39855811  speedup is 46.65
    building file list ... done
    Flutter.framework/
    Flutter.framework/Flutter
    Flutter.framework/Info.plist
    Flutter.framework/icudtl.dat
    Flutter.framework/_CodeSignature/
    Flutter.framework/_CodeSignature/CodeResources

    sent 98828329 bytes  received 120 bytes  197656898.00 bytes/sec
    total size is 98815875  speedup is 1.00
    /Users/eb/AndroidStudioProjects/playpadel/build/ios/Debug-iphonesimulator/Runner.app/Frameworks/App.framework/App: signed bundle
    with Mach-O universal (x86_64 arm64) [io.flutter.flutter.app]
    /Users/eb/AndroidStudioProjects/playpadel/build/ios/Debug-iphonesimulator/Runner.app/Frameworks/Flutter.framework/Flutter: replacing
    existing signature
    /Users/eb/AndroidStudioProjects/playpadel/build/ios/Debug-iphonesimulator/Runner.app/Frameworks/Flutter.framework/Flutter: signed
    bundle with Mach-O universal (armv7 x86_64 arm64) [io.flutter.flutter]
    2020-08-20 16:39:35.038 defaults[1882:18363] 
    The domain/default pair of
    (/Users/eb/AndroidStudioProjects/playpadel/build/ios/Debug-iphonesimulator/Runner.app/Frameworks/App.framework/flutter_assets/ios/Fl
    utter/App.framework/Info.plist, CFBundleExecutable) does not exist
    fatal error: lipo: can't map input file:
    /Users/eb/AndroidStudioProjects/playpadel/build/ios/Debug-iphonesimulator/Runner.app/Frameworks/App.framework/flutter_assets/ios/Flu
    tter/App.framework/ (Invalid argument)
    fatal error: lipo: can't map input file:
    /Users/eb/AndroidStudioProjects/playpadel/build/ios/Debug-iphonesimulator/Runner.app/Frameworks/App.framework/flutter_assets/ios/Flu
    tter/App.framework/ (Invalid argument)
    Failed to extract x86_64 for
    /Users/eb/AndroidStudioProjects/playpadel/build/ios/Debug-iphonesimulator/Runner.app/Frameworks/App.framework/flutter_assets/ios/Flu
    tter/App.framework/. Running lipo -info:
    fatal error: lipo: can't map input file:
    /Users/eb/AndroidStudioProjects/playpadel/build/ios/Debug-iphonesimulator/Runner.app/Frameworks/App.framework/flutter_assets/ios/Flu
    tter/App.framework/ (Invalid argument)
    Command PhaseScriptExecution failed with a nonzero exit code
    note: Using new build system
    note: Building targets in parallel
    note: Planning build
    note: Constructing build description
    warning: Capabilities for Signing & Capabilities may not function correctly because its entitlements use a placeholder team ID. To
    resolve this, select a development team in the Runner editor. (in target 'Runner' from project 'Runner')

Could not build the application for the simulator.
Error launching application on iPhone SE (2nd generation).

``` 

</details> 

###flutter doctor -v
eb@erics-MacBook-Pro playpadel % flutter doctor -v
[✓] Flutter (Channel stable, 1.20.2, on Mac OS X 10.15.6 19G2021, locale en-IT)
    • Flutter version 1.20.2 at /Users/eb/flutter
    • Framework revision bbfbf1770c (7 days ago), 2020-08-13 08:33:09 -0700
    • Engine revision 9d5b21729f
    • Dart version 2.9.1

 
[✓] Android toolchain - develop for Android devices (Android SDK version 30.0.2)
    • Android SDK at /Users/eb/Library/Android/sdk
    • Platform android-30, build-tools 30.0.2
    • Java binary at: /Applications/Android
      Studio.app/Contents/jre/jdk/Contents/Home/bin/java
    • Java version OpenJDK Runtime Environment (build 1.8.0_242-release-1644-b3-6222593)
    • All Android licenses accepted.

[✓] Xcode - develop for iOS and macOS (Xcode 11.6)
    • Xcode at /Applications/Xcode.app/Contents/Developer
    • Xcode 11.6, Build version 11E708
    • CocoaPods version 1.9.3

[✓] Android Studio (version 4.0)
    • Android Studio at /Applications/Android Studio.app/Contents
    • Flutter plugin version 48.1.2
    • Dart plugin version 193.7361
    • Java version OpenJDK Runtime Environment (build 1.8.0_242-release-1644-b3-6222593)

[✓] Connected device (1 available)
    • iPhone SE (2nd generation) (mobile) • EC8058CA-3A91-4C6A-823F-6D9AD2510BD1 • ios •
      com.apple.CoreSimulator.SimRuntime.iOS-13-6 (simulator)

• No issues found!

**Comment by ioplaypadelapp** [comment]
> Hi @ioplaypadelapp
> Can you please provide a complete reproducible minimal code sample repository
> Thank you

I'm sorry I can't do that. After the issue I donwgraded to flutter 1.16.1 and  everything is working fine. 
Intructions I followed:
- open terminal in flutter project folder
- flutter version v1.16.1
- move ios folder to bin (using finder)
- flutter clean
- flutter create .
- xcode optional stuff: change bundle id, add capabilities, add google services, etc
- flutter run

**Comment by TahaTesser** [comment]
@ioplaypadelapp @mateobadillo @ubxty  @hauphidao 
This seems to be fixed in https://github.com/flutter/flutter/issues/57210, which is not landed into `stable` yet

Can you upgrade to latest `dev` and try again
`flutter channel dev`
`flutter upgrade --force`
`flutter doctor -v`

**Comment by SametSahin10** [comment]
> @ioplaypadelapp @mateobadillo @ubxty @hauphidao
> This seems to be fixed in #57210, which is not landed into `stable` yet
> 
> Can you upgrade to latest `dev` and try again
> `flutter channel dev`
> `flutter upgrade --force`
> `flutter doctor -v`

Switching to `dev` channel solved it for now.

**Comment by ioplaypadelapp** [comment]
> > @ioplaypadelapp @mateobadillo @ubxty @hauphidao
> > This seems to be fixed in #57210, which is not landed into `stable` yet
> > Can you upgrade to latest `dev` and try again
> > `flutter channel dev`
> > `flutter upgrade --force`
> > `flutter doctor -v`
> 
> Switching to `dev` channel solved it for now.

Same for me

**Comment by TahaTesser** [comment]
Hi @ioplaypadelapp
Thanks for confirming @SametSahin10 
This will most likely be cherry-picked for 1.20.3 stable  https://github.com/flutter/flutter/issues/57210#issuecomment-674159005
Closing, as fixed. If you disagree, please write in the comments and I will reopen it.
Thank you

**Comment by moux2003** [comment]
Got the exact same error when going with Flutter 1.22
Rolled back to 1.20.3 and everything is ok, can't reproduce with a new project and can't afford to recreate ios project as it has too much configuration (deep links, i18n, multiple capabilities, firebase services etc...)

**Comment by TahaTesser** [comment]
Could everyone who still has this problem please file a new issue with the exact description of what happens, logs, and the output of `flutter doctor -v`.
All system setups can be slightly different, so it's always better to open new issues and reference related issues.

**Comment by moux2003** [comment]
@TahaTesser sorry it was my bad.. This errors came but the real error I found in logs was the sksl file engine was outdated.
Sorry again.
